src/playercomponent.js

```diff
@@ -162,18 +161,26 @@ function PlayerComponent(
 
   function eventCallback(mediaState) {
     switch (mediaState) {
-      case MediaState.PLAYING:
+      case MediaState.PLAYING: {
         onPlaying()
         break
-      case MediaState.PAUSED:
+      }
+      case MediaState.PAUSED: {
         onPaused()
         break
-      case MediaState.WAITING:
+      }
+      case MediaState.WAITING: {
         onBuffering()
         break
-      case MediaState.ENDED:
+      }
+      case MediaState.ENDED: {
         onEnded()
         break
+      }
+
+      default: {
+        break
```

Comment:
What are thoughts on adding some kind of error/telemetry for an unrecognised media state? I'm not suggesting it be part of this ticket. Moreso one to raise as a new ticket?